docs/src/guides/hub-entities.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Hub Entities
+navTitle: Hub Entities
+description: Core Entities in ArcGIS Hub
+order: 3
+group: 2-concepts
+---
+
+# Hub Entities
+
+While ArcGIS Hub acts as an integration point for the entire ArcGIS ecosystem, at it's core are 5 main entities:
+
+- Hub Sites
+- Hub Pages
+- Hub Initiatives
+- Hub Projects
+- Hub Events
+
+All other ArcGIS content is handled generically as "Hub Content"
```

Comment:
You probably don't want to get into any of this here, but content is more than "other". Sometimes a hub page is treated as content, etc.